lib/workers/repository/process/lookup/filter-checks.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { getConfig, getName, mocked } from '../../../../../test/util';
+import { Release } from '../../../../datasource';
+import { clone } from '../../../../util/clone';
+import * as _dateUtil from '../../../../util/date';
+import * as allVersioning from '../../../../versioning';
+import { filterInternalChecks } from './filter-checks';
+import { LookupUpdateConfig, UpdateResult } from './types';
+
+jest.mock('../../../../util/date');
+
+const dateUtil = mocked(_dateUtil);
+
+let config: Partial<LookupUpdateConfig & UpdateResult>;
+
+const versioning = allVersioning.get('semver');
+
+const releases: Release[] = [
+  {
+    version: '1.0.1',
+    releaseTimestamp: '2021-01-01T00:00:01.000Z',
+  },
+  {
+    version: '1.0.2',
+    releaseTimestamp: '2021-01-03T00:00:00.000Z',
+  },
+  {
+    version: '1.0.3',
+    releaseTimestamp: '2021-01-05T00:00:00.000Z',
+  },
+  {
+    version: '1.0.4',
+    releaseTimestamp: '2021-01-07T00:00:00.000Z',
+  },
+];
+
+describe(getName(), () => {
+  let sortedReleases: Release[];
+  beforeEach(() => {
+    config = getConfig();
+    config.currentVersion = '1.0.0';
+    sortedReleases = clone(releases);
+    jest.resetAllMocks();
+    dateUtil.getElapsedDays.mockReturnValueOnce(3);
+    dateUtil.getElapsedDays.mockReturnValueOnce(5);
+    dateUtil.getElapsedDays.mockReturnValueOnce(7);
+    dateUtil.getElapsedDays.mockReturnValueOnce(9);
+  });
+
+  describe('.filterInternalChecks()', () => {
+    it('returns latest release if internalChecksFilter=none', () => {
+      const res = filterInternalChecks(
+        config,
+        versioning,
+        'patch',
+        sortedReleases
+      );
+      expect(res).toMatchSnapshot();
+      expect(res.pendingChecks).toHaveLength(0);
+      expect(res.pendingReleases).toHaveLength(0);
+      expect(res.release.version).toEqual('1.0.4');
+    });
+    it('returns non-pending latest release if internalChecksFilter=flexible and none pass checks', () => {
```

Comment:
```suggestion

    it('returns non-pending latest release if internalChecksFilter=flexible and none pass checks', () => {
```

Some newlines between tests for better readability?